engine/class_modules/sc_shaman.cpp

```diff
@@ -7461,11 +7466,19 @@ struct earthquake_overload_damage_t : public earthquake_damage_base_t
   earthquake_overload_damage_t( shaman_t* player, earthquake_base_t* parent ) :
     earthquake_damage_base_t( player, "earthquake_overload_damage", player->find_spell( 298765 ), parent )
   {
-    // Earthquake modifier is hardcoded rather than using effects, so we set the modifier here
-    spell_power_mod.direct = 0.3195 * player->talent.mountains_will_fall->effectN( 1 ).percent();
-    // TODO: implement spellpower coefficient extracttion from spell variable
-    // auto spell_desc = player->dbc->spell_desc_vars( this->data().id() );
-    // spell_desc.desc_vars()
+
```

Comment:
Should this still have a default? As far as I know, this code will now make Earthquake / overload do 0 damage if the regex fails. One option would be to add (an expansion specific) default value that can be updated manually, so 0.3195 in tww. Another option would be to assert if the coefficient is 0, this should trigger CI if it gets changed.